packages/core/src/domain/user/user.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id?: string
+  email?: string
+  name?: string
+  [key: string]: unknown
+}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏as hinted by compatibility tests, this is a breaking change